package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/paginate-anything.js",
   "private": false,
   "scripts": {
-	"test": "jasmine-node spec/"
+	"test": "node_modules/jasmine-node/bin/jasmine-node spec"
```

Comment:
Not everyone has jasmine-node installed globally. This will allow `npm test` to run.